ios/Sources/AutoLayoutView.swift

```diff
@@ -266,6 +289,7 @@ import UIKit
     }
 
     private func footer() -> UIView? {
-        return superview?.subviews.first(where:{($0 as? CellContainer)?.index == -1})
+        // TODO: Investigate hierarchy here
```

Comment:
What exactly do you mean?

Reply:
That was a reminder for me 😅. I've updated it now.